runtime/binding-http/src/test/java/io/aklivity/zilla/runtime/binding/http/internal/config/HttpOptionsConfigAdapterTest.java

```diff
@@ -37,9 +38,13 @@
 import org.junit.Test;
 
 import io.aklivity.zilla.runtime.binding.http.config.HttpOptionsConfig;
+import io.aklivity.zilla.runtime.binding.http.config.HttpRequestConfig;
 import io.aklivity.zilla.runtime.binding.http.config.HttpVersion;
 import io.aklivity.zilla.runtime.binding.http.internal.types.String16FW;
 import io.aklivity.zilla.runtime.binding.http.internal.types.String8FW;
+import io.aklivity.zilla.runtime.engine.internal.validator.config.AvroValidatorConfig;
+import io.aklivity.zilla.runtime.engine.internal.validator.config.CatalogedConfig;
+import io.aklivity.zilla.runtime.engine.internal.validator.config.StringValidatorConfig;
```

Comment:
In preparation for hoisting out the validators, this should be using a `TestValidator` instead of specific implementations.

The string validator and avro validator config tests belong closer to those implementations.

Reply:
This is coming in PR #440.